Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -224,6 +263,22 @@ extension StripeCardReaderService: DiscoveryDelegate {
 }
 
 
+// MARK: - ReaderDisplayDelegate.
+extension StripeCardReaderService: ReaderDisplayDelegate {
+    public func terminal(_ terminal: Terminal, didRequestReaderInput inputOptions: ReaderInputOptions = []) {
+        print("==== did request reader input")
+        print(inputOptions)
+        print(inputOptions.rawValue)
+        print("//// did request reader input")
+    }
+
+    public func terminal(_ terminal: Terminal, didRequestReaderDisplayMessage displayMessage: ReaderDisplayMessage) {
```

Comment:
Maybe add a comment that didRequestReaderDisplayMessage means the reader has a message for us to display to the buyer